docs/content/node/getting-started/quickstart/data/data.md

```diff
@@ -19,4 +19,47 @@ endswithURL: >
 expensesPath1: examples/quickstart/polar/expenses-01-nodejs.polar
 expensesPath2: examples/quickstart/polar/expenses-02-nodejs.polar
 isAllowed: isAllowed
+installation_new: |
+    ```bash
+    npm install --save oso
+    ```
+    Or find the Oso package at [http://pypi.python.org/pypi/oso/](http://pypi.python.org/pypi/oso/).
+import: import
+import_code: |
+    ```js
+    const { Oso } = require('oso')
+    oso = Oso()
+    oso.enableRoles()
+    ```
+example_app: an Express
+load_policy: |
+    ```js
+    oso.loadFile("authorization.polar")
+    ```
+get_roles: getRoles
+classes: JavaScript classes
+objects: JavaScript objects
+methods: JavaScript methods
+register_classes: |
+    ```js
+    oso.registerClass(Page)
+    oso.registerClass(User)
+    ```
+app_code: |
+    ```js
+    const app = express()
+    const port = 3000
+
+    app.get('/page/:pageNum', async (req, res) => {
+        const pageNum = req.params.pageNum
+        const page = Page.getPage(pageNum)
+        const user = User.getCurrentUser()
+        if (await oso.isAllowed(user, "read", page)) {
+            res.send(`<h1>A Page</h1><p>this is page ${page.pagenum}</p>`)
```

Comment:
should be `page.pageNum` (or, less repetitively, `page.num`) to match the casing convention elsewhere in this snippet

Reply:
I want all of these to be literal includes once the new quickstarts land. We should do that as an immediate follow up.

Reply:
Should do this before